src/transformers/models/dbrx/modeling_dbrx.py

```diff
@@ -613,7 +613,7 @@ def forward(self, hidden_states: torch.Tensor) -> Tuple[torch.Tensor, torch.Tens
         top_weights, top_experts = torch.topk(weights, self.moe_top_k, dim=-1)
 
         top_weights_scale = (
-            torch.norm(top_weights, p=self.moe_normalize_expert_weights, dim=-1, keepdim=True)
+            torch.linalg.norm(top_weights, p=self.moe_normalize_expert_weights, dim=-1, keepdim=True)
```

Comment:
Hi @cyyever, I think the signature of `torch.linalg.norm` is different and doesn't have a `p` kwarg!

Reply:
Revert that change